package.json

```diff
@@ -62,8 +63,8 @@
     "stylelint": "^13.0.0",
     "stylelint-config-recommended": "^3.0.0",
     "ts-node": "^9.0.0",
-    "wtfnode": "^0.8.4",
-    "typescript": "^4.1.3"
+    "typescript": "^4.1.3",
+    "wtfnode": "^0.8.4"
```

Comment:
I wonder if this was a manual change or something done automatically? Not particularly an issue, just curious :eyes:

Reply:
I think that happens when you add a new dependency to the top? Not sure, I did see it in one of Asher's PRs

Reply:
Yeah `yarn` resorts the dependencies. :smile:

Reply:
The real question is how it got out of order in the first place lol

Reply:
LOL I have no idea - could have been me 😂

Reply:
Haha idk I say we just blame `yarn`.